### DISCUSSION TITLE
Windows 11 help

### DISCUSSION THREAD

**Discussion by tc-kirby** [opensteno/plover#1617] [Q&A and Support] [answered]
Hello I'm just getting started with steno and I've built myself a Picosteno and installed Plover on a Windows 11 machine.

I've got up to the point of trying to write 'Hello World' according to the instructions on the Wiki, but it's going badly. I am sure Plover is enabled and I am pressing the right key combinations, but it keeps giving me different output each time. Sometimes I press keys and it does nothing, and then I move the mouse and then the output "comes through".

I've tried running Plover as admin, I've tried running version 3, and I've tried installing Plover on a different Win 11 machine, but none of those things work.

This happens whether I use my Picosteno or just a regular keyboard.

I also have an Ubuntu machine, but I understand that Plover is incompatible with Wayland, so at the moment unless I install an old/difficult-to-use Linux distro, I have no way of even beginning to get started with stenography.

Please can someone help me?

Thanks

Tom

**Comment by user202729** [comment]
not sure what's going on. try the following

* open the paper tape and compare the strokes you see on the tape with the desired strokes.
* press just one key at a time. see if it works.
* try switching to NKRO mode as well as serial mode.
* If it does but multiple keys at the same time doesn't, maybe NKRO issue.

**Comment by tc-kirby** [comment, marked as answer]
Turns out the problem was that I was using Notepad. In Word, everything works as expected!

I'm using NKRO. Not sure how I can use serial with just a USB cable to the Pico... difficult to find instructions on how to configure Plover correctly for a serial connection with Picosteno.
